packages/admin/blocks-admin/src/blocks/VimeoVideoBlock.tsx

```diff
@@ -0,0 +1,52 @@
+import { Field, FinalFormInput, FinalFormSwitch } from "@comet/admin";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { VimeoVideoBlockData } from "../blocks.generated";
+import { BlocksFinalForm } from "../form/BlocksFinalForm";
+import { createCompositeBlock } from "./factories/createCompositeBlock";
+import { createCompositeSettings } from "./helpers/composeBlocks/createCompositeSettings";
+import { BlockCategory } from "./types";
+
+export const VimeoVideoBlock = createCompositeBlock(
+    {
+        name: "VimeoVideo",
+        displayName: <FormattedMessage id="blocks.vimeoVideo" defaultMessage="Video (Vimeo)" />,
+        category: BlockCategory.Media,
+        blocks: {
+            settings: {
+                block: createCompositeSettings<Pick<VimeoVideoBlockData, "vimeoIdentifier" | "autoplay">>({
+                    defaultValues: { vimeoIdentifier: "", autoplay: false },
+                    AdminComponent: ({ state, updateState }) => {
+                        return (
+                            <BlocksFinalForm onSubmit={updateState} initialValues={state}>
+                                <Field
+                                    name="vimeoIdentifier"
+                                    label={<FormattedMessage id="blocks.vimeoVideo.vimeoIdentifier" defaultMessage="Vimeo URL or Vimeo Video ID" />}
+                                    type="text"
+                                    component={FinalFormInput}
+                                    fullWidth
+                                />
+
+                                <Field
+                                    name="autoplay"
```

Comment:
DAMVideoBlock also has loop and showControls, does it makes sense to add it here as well?

Reply:
I guess in general it makes sense to streamline all video blocks so that they behave pretty similar 🤔 Should I add those controls + aspect ratio then as fields?

Reply:
I'd say yes

Reply:
Done